cmake/embed/embed_clang.cmake

```diff
@@ -82,7 +52,7 @@ endif()
 set(CLANG_CONFIGURE_FLAGS
     -Wno-dev
     -DLLVM_TARGETS_TO_BUILD=BPF
-    -DCMAKE_BUILD_TYPE=${EMBEDDED_BUILD_TYPE}
+    -DCMAKE_BUILD_TYPE=MinSizeRel
```

Comment:
I don't think there is any reason to build RelWithDebinfo for embedded, it bloats the size on Github Actions a lot, and only 14GB disk is available.